todo_rxbloc_app/lib/feature_manage_todo/views/manage_todo.dart

```diff
@@ -0,0 +1,139 @@
+                              cursorBehaviour:
+                                  RxTextFormFieldCursorBehaviour.preserve,
+                              state: (bloc) => bloc.states.validateTitle,
+                              showErrorState: (bloc) => bloc.states.showErrors,
+                              onChanged: (bloc, task) => bloc.events.setTitle(
+                                task.trim(),
+                              ),
+                              builder: (fieldState) => TextFormField(
+                                autofocus: widget.todo == null ? true : false,
+                                controller: fieldState.controller,
+                                decoration:
+                                    fieldState.decoration.copyWithDecoration(
+                                  InputDecoration(
+                                    hintText: context.l10n.featureAddTodo.task,
+                                  ),
+                                ),
+                                style: Theme.of(context).textTheme.headline5,
+                              ),
+                            ),
+                            SizedBox(
+                                height: context.designSystem.spacing.space25),
+                            TextFormField(
+                              onChanged: (description) => context
+                                  .read<ManageTodoBlocType>()
+                                  .events
+                                  .setDescription(description.trim()),
+                              controller: noteController,
+                              decoration: InputDecoration(
+                                hintText: context.l10n.featureAddTodo.note,
+                              ),
+                              maxLines: 10,
+                              style: Theme.of(context).textTheme.subtitle1,
+                            ),
+                          ],
+                        ),
+                      )
+                    : const Center(
+                        child: CircularProgressIndicator(),
+                      ),
+              ),
+              floatingActionButton: FloatingActionButton(
+                child: widget.todo != null
+                    ? Icon(context.designSystem.icons.edit)
+                    : Icon(context.designSystem.icons.plusSign),
+                onPressed: () => widget.todo != null
+                    ? context.read<ManageTodoBlocType>().events.updateTodo()
+                    : context.read<ManageTodoBlocType>().events.saveTodo(),
+              ),
+            ),
+          ),
+        ],
+      ),
+    );
+  }
+
+  @override
+  void dispose() {
+    noteController.dispose();
+    super.dispose();
+  }
+
+  Widget _successToastMessages(BuildContext context) =>
+      RxBlocListener<ManageTodoBlocType, NewTodoEnum>(
+        state: (bloc) => bloc.states.newTodo,
+        listener: (context, state) {
```

Comment:
As discussed it's good practice to keep the widget as part of the build method, where for the callbacks such as `listener: (context, state)` to have a private method